util/ubuntu-dependencies

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This script installs streisand's dependencies on an Ubuntu 16.04
+# system; Debian jessie has been lightly tested. It installs system
+# packages for binary dependencies, and installs the rest in the
+# python directories in $HOME/.local.
+
+# We default to a regular, interactive upgrade:
+quiet=
+
+# It's safe to run this script multiple times.
+#
+# Please don't paste this directly into an interactive shell; if you
+# do, I will instantly close any bugs you file.
+#
+# If you're brave: this script can proceed mostly without interaction if:
+#
+# 1) "sudo echo" will succeed (i.e., you have root without a prompt now)
+#
+# 2) uncomment the following line:
+
+# quiet='--yes'
+
+# If you *really* want no interaction (for example, overwriting local
+# config files without prompting), then uncomment this:
+# export DEBIAN_FRONTEND=noninteractive
+
+### No options below this line. ###
+
+set -e
+set -x
+
+function our_pip_install () {
+    pip install --user --upgrade "$@"
+}
+
+# Hopefully, the blank lines following these statements will answer "yes" if
```

Comment:
I think its better to not try and support this case at all and hopefully fail fast, sooner. It also makes the script less readable.